server/notification-providers/discord.js

```diff
@@ -33,6 +33,21 @@ class Discord extends NotificationProvider {
                 return okMsg;
             }
 
+            const embedFields = [
+                {
+                    name: "Service Name",
+                    value: monitorJSON["name"],
+                },
+                {
+                    name: `Time (${heartbeatJSON["timezone"]})`,
+                    value: heartbeatJSON["localDateTime"],
+                },
+                {
+                    name: "Error",
+                    value: heartbeatJSON["msg"] == null ? "N/A" : heartbeatJSON["msg"],
```

Comment:
Let's use this instead

```suggestion
                    value: msg,
```

Reply:
Scratch that. Looking at https://github.com/louislam/uptime-kuma/pull/5181#issuecomment-2403721162 this looks bad (👀).

The original author had a point..

Let's do the following instead:

```suggestion
                    value: heartbeatJSON["msg"] ?? msg,
```